developer/kmcompx/src/meson.build

```diff
@@ -0,0 +1,102 @@
+#
+# Keyman is copyright (C) SIL International. MIT License.
+#
+# Cross platform build script to compile kmcompx
+#
+
+defns = ['-DKMN_KBP_EXPORTING']
+version_res = []
+
+# TODO: move these shared defines to a resources/meson folder?
+if compiler.get_id() == 'gcc' or compiler.get_id() == 'clang'
+  warns = [
+     '-Wall',
+     '-Wctor-dtor-privacy',
+     '-Wdouble-promotion',
+     '-Wendif-labels',
+     '-Wextra',
+     '-Wno-unknown-pragmas',
+     '-Wno-missing-field-initializers',
+     '-Wnon-virtual-dtor',
+     '-Wshadow'
+  ]
+  flags = [
+    '-fvisibility=hidden',
+    '-fvisibility-inlines-hidden'
```

Comment:
You can instead add the kwarg `gnu_symbol_visibility: 'inlineshidden'` to the target. Meson will check whether the current compiler supports both these flags, then add both of them if supported.

Reply:
Unfortunately that's only available in 0.48